user_interaction/static/themes/q-theme.js

```diff
@@ -1,9 +1,9 @@
+"use strict";
+
 // Modify the DOM
-$( document ).ready(function() {
-    console.debug('hello!')
-    let html = $('body main').html()
+$(document).ready(function () {
+    console.debug('hello!');
     // Replace 'k' and 'c' by 'q' (leave html between < and > alone)
```

Comment:
Strictly speaking the complicated regex _should_ not be needed anymore. But it's scary to touch, and doesn't hurt anyone.